### PR TITLE
Organize things a bit

### PR DESCRIPTION
**Pull request by jtaleric**
Previously there was just a collection of scripts. This commit is to
clean things up.

### CODE REVIEW

File: workloads/REDME.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Workload scripts
```
Comment (rsevilla87) [inline workloads/REDME.md]: Are we missing more tests in this list?

File: workloads/network-perf/run_serviceip_network_test_fromgit.sh [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+client=""
+pin=false
+if [[ $(oc get nodes | grep worker | wc -l) -gt 1 ]]; then
+  server=$(oc describe nodes/$(oc get nodes | grep worker | tail -1 | awk '{print $1}') | grep hostname | awk -F= '{print $2}')
+  client=$(oc describe nodes/$(oc get nodes | grep worker | head -1 | awk '{print $1}') | grep hostname | awk -F= '{print $2}')
+  pin=true
+fi
+
+# Create Service Account with View privileges for backpack
+oc delete ClusterRoleBinding/backpack-view
```
Comment (rsevilla87) [inline workloads/network-perf/run_serviceip_network_test_fromgit.sh]: We can use ripsaw's YAML files to create ClusterRoleBinding and serviceAccount, as these ones don't inherit any parameters

File: workloads/network-perf/run_serviceip_network_test_fromgit.sh [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+git clone http://github.com/cloud-bulldozer/ripsaw /tmp/ripsaw
+oc apply -f /tmp/ripsaw/deploy
+oc apply -f /tmp/ripsaw/resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml
+oc apply -f /tmp/ripsaw/resources/operator.yaml
+
+server=""
+client=""
+pin=false
+if [[ $(oc get nodes | grep worker | wc -l) -gt 1 ]]; then
+  server=$(oc describe nodes/$(oc get nodes | grep worker | tail -1 | awk '{print $1}') | grep hostname | awk -F= '{print $2}')
```
Comment (rsevilla87) [inline workloads/network-perf/run_serviceip_network_test_fromgit.sh]: Something like `oc get nodes -l  node-role.kubernetes.io/worker -o jsonpath='{range .items[*]}{ .metadata.labels.kubernetes\.io/hostname}{"\n"}{end}'`can save likely mistakes and API calls

File: workloads/network-perf/run_serviceip_network_test_fromgit.sh [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+  client=$(oc describe nodes/$(oc get nodes | grep worker | head -1 | awk '{print $1}') | grep hostname | awk -F= '{print $2}')
```
Comment (rsevilla87) [inline workloads/network-perf/run_serviceip_network_test_fromgit.sh]: Same here :)

File: workloads/REDME.md (hunk 1 above)
Comment (rsevilla87) [reply workloads/REDME.md]: REDME.md?

File: workloads/network-perf/run_serviceip_network_test_fromgit.sh (hunk 2 above)
Comment (jtaleric) [reply workloads/network-perf/run_serviceip_network_test_fromgit.sh]: Hm, not sure about that? @dry923 thoughts?

File: workloads/network-perf/run_serviceip_network_test_fromgit.sh (hunk 4 above)
Comment (jtaleric) [reply workloads/network-perf/run_serviceip_network_test_fromgit.sh]: Great suggestion!

File: workloads/network-perf/run_serviceip_network_test_fromgit.sh (hunk 2 above)
Comment (jtaleric) [reply workloads/network-perf/run_serviceip_network_test_fromgit.sh]: Ah ha! I see you point now. Fixing!

**Comment by mohit-sheth** [comment]
LGTM

**Comment by rsevilla87** [comment]
LGTM

**Pull request merged**